lrlex/src/lib/parser.rs

```diff
@@ -379,51 +379,85 @@ impl<StorageT: TryFrom<usize>> LexParser<StorageT> {
             ///
             /// Meaning: The character 'c', unchanged.
             fn unescape(re: Cow<str>) -> Cow<str> {
+                // POSIX lex has two layers of escaping, there are escapes for the regular expressions themselves
+                // And the escapes which get handled by lex.  We can find what the `regex` crate considers
+                // an escaped regular expression meta_character which requires escaping. through `is_meta_character`.
+                //
+                // We generally want to avoid sending Regex an escaped character which it does not consider a
```

Comment:
Why "generally"?

Reply:
Removed the word, and expanded on the alternate situation in the comment in https://github.com/softdevteam/grmtools/pull/333/commits/819f4e9b626ae3e7f5d631757611c349a0013953.